test/test_graphic3d_rendering_params.py

```diff
@@ -0,0 +1,438 @@
+# This file is part of pyOCCT which provides Python bindings to the OpenCASCADE
+# geometry kernel.
+#
+# Copyright (C) 2016-2018  Laughlin Research, LLC (info@laughlinresearch.com)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+import unittest
+
+from OCCT.Graphic3d import (Graphic3d_RenderingParams,
+        Graphic3d_RM_RASTERIZATION, Graphic3d_RM_RAYTRACING,
+        Graphic3d_RTM_BLEND_UNORDERED, Graphic3d_RTM_BLEND_OIT,
+        Graphic3d_StereoMode_QuadBuffer, Graphic3d_StereoMode_SideBySide,
+        Graphic3d_ToneMappingMethod_Disabled,
+        Graphic3d_ToneMappingMethod_Filmic)
+
+
+class TestGraphic3dRenderingParamsDataFields(unittest.TestCase):
```

Comment:
Can you put this in a single file called `test_graphic3d.py` and then have `class TestGraphic3dRenderingParams(unittest.TestCase)`? I think over time we'll build up test modules for each pyOCCT module, and then define classes inside the module for testing specific classes. What do you think?

Reply:
That sounds like a good arrangement. Changes made.